Core/GameEngine/Source/Common/System/RAMFile.cpp

```diff
@@ -104,7 +104,6 @@
 RAMFile::RAMFile()
 : m_size(0),
 	m_data(NULL),
-//Added By Sadullah Nader
 //Initializtion(s) inserted
```

Comment:
The other nearby useless comments can also be removed.

```cpp
//Initializtion(s) inserted

//
```

Reply:
Ok I made another pass, removed a lot more comments